src/fidesops/service/masking/masking_processor.py

```diff
@@ -19,5 +19,6 @@ def mask(value: str, masking_strategies: List[PolicyMaskingSpec]) -> str:
         strategy = masking_strategy_factory.get_strategy(
             strategy.strategy, strategy.configuration
         )
-        masked_value = strategy.mask(masked_value)
+        # fixme: generate secrets, but what privacy request id to use?
```

Comment:
Here's another place where `mask()` is called, but without context of privacy request. This bit of code doesn't seem used at the moment, so what is it for?

Reply:
Probably just some legacy code from earlier testing, before we had the request execution working? If it's unused I'd recommend deleting it and refactoring the tests.